test/core/client_channel/BUILD

```diff
@@ -33,6 +34,69 @@ grpc_cc_test(
     ],
 )
 
+grpc_cc_library(
+    name = "client_channel_test_lib",
```

Comment:
Does yodel always require two targets for each test?  Can we maybe have a macro to wrap them up into a single target for this seemingly common case?

Reply:
that seems reasonable yeah :) done